linux/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(OpenSupaplex LANGUAGES C)
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED YES)
+
+add_compile_definitions(HAVE_SDL2)
+
+add_library(common STATIC
+	../src/supaplex.c
+	../src/animations.c
+	../src/buttonBorders.c
+	../src/commandLineParser.c
+	../src/conditionals.c
+	../src/config.c
+	../src/demo.c
+	../src/file.c
+	../src/globals.c
+	../src/graphics.c
+	../src/logging.c
+	../src/input.c
+	../src/menu.c
+	../src/utils.c
+	../src/savegame.c
+	../src/sdl_common/system.c
+	../src/sdl2/controller.c
+	../src/sdl2/keyboard.c
+	../src/sdl2/touchscreen.c
+	../src/null/virtualKeyboard.c
+	../src/lib/ini/ini.c
+)
+target_link_libraries(common SDL2 SDL2_mixer m)
+
+add_executable(opensupaplex
+	../src/sdl_common/audio.c
+	../src/sdl2/video.c
+)
+target_link_libraries(opensupaplex common)
+
+add_executable(opensupaplex-test EXCLUDE_FROM_ALL
+	../src/null/audio.c
+	../src/null/video.c
+)
+target_link_libraries(opensupaplex-test common)
```

Comment:
Why using CMake instead of a plain `Makefile`? I don't have anything against CMake (mainly because I'm not more than a newbie), but the Makefile seemed much much simpler and with it you don't need a SDL2 submodule to make it work 🤔

Reply:
This is pretty simple. Actually, I did this to cleanup previous distinction between Makefile and Makefile.tests, which were *mostly* similar, and I dislike duplicate code, where it's not clear what the difference is and why.

Here, it's obvious what is the difference between two targets.

Also, I'm preparing for the next change to fix #7 and I think it's a bit easier this way than with Makefile.

Reply:
Yes, I agree with that, but that was a project structure problem because the tests stuff was in the `linux` subfolder and it shouldn't 😓 So I had some leftovers that didn't really belong to the `linux` subfolder. That's my fault, sorry about that.

Without them (after moving that stuff away in da367ba8b5808f4019ed1b4ec441fc66654798e0) there is no need for `Makefile.tests` or making distinctions with CMake.

As long as the Makefile is simple and can handle easily the complexity of this project (which should be pretty pretty simple 🤞) I'd rather stick to `Makefile`

Reply:
Okay. Why do you use cmake for vita?

Moving the file to a different folder doesn't remove duplication, it's still two very similar makefiles instead of one.

Reply:
> Why do you use cmake for vita?

Because that's the template VitaSDK provides 😅 The whole VitaSDK seems to be designed to be used with CMake, while, as of today, SDKs for other platforms (Wii, PSP, Nintendo Switch, 3DS, PS3, Wii U) prefer Makefiles. I guess Makefiles can be used with VitaSDK and CMake can be used with the other platforms, but it seems to be less convenient and didn't want to get into that fight, because I already had a lot on my plate with all the SDL fixes and that 😂 

> Moving the file to a different folder doesn't remove duplication

That's right and I agree. My intention keeping that stuff duplicated _for now_ is making sure I don't end up with complex scripts / Makefiles that support many different platforms, and hoping that in the end, if having many (partially duplicated) Makefiles is really problem, there is a better chance to see the big picture and think about what's the best way of abstracting and de-duplicating all that without causing a mess.

Maybe that's where CMake would come in?

Meanwhile, I'd rather stick to the simplest approach until it becomes an issue, as long as it doesn't create a tremendous pain (which I don't think is the case 😳).